.github/workflows/ci.yaml

```diff
@@ -55,6 +55,7 @@ jobs:
           -DMIGRAPHX_ENABLE_GPU=On \
           -DMIGRAPHX_ENABLE_CPU=On \
           -DMIGRAPHX_ENABLE_FPGA=On \
+          -DMIGRAPHX_ENABLE_MLIR=On \
```

Comment:
Should this be off if we're disabling mlir?

Reply:
No, I want tidy to be enabled so we can still catch tidy issues.

Reply:
all good. I saw your previous message after I wrote that. Should be fine